libs/core/.storybook/preview.js

```diff
@@ -25,5 +25,21 @@ export const parameters = {
   docs: {
     extractArgTypes,
     extractComponentDescription,
+  },
+  options: {
+    storySort: {
+      order: [
+        'Welcome',
+        'Resources',
+        'Foundations',
+        [
+          'Design Principles',
+          'Colors',
+          'Typography'
+        ],
+        'Components',
+        'Get Support'
+      ],
```

Comment:
ℹ️  This assists with creating an order of the MDX files in the left Sidebar. 

Do you foresee this as overhead should we want to change the order?

Reply:
> This looks awesome!
> 
> I'm just seeing a few little alignment issues in the sidebar, looks like the change in font-family broke the alignment between text and the icon a bit:
> 
> Before	After
> <img alt="CleanShot 2023-02-14 at 11 06 42@2x" width="183" src="https://user-images.githubusercontent.com/791670/218865423-70eababd-f2f6-45e8-9367-2fcf7a406498.png">	<img alt="CleanShot 2023-02-14 at 16 22 54@2x" width="149" src="https://user-images.githubusercontent.com/791670/218866161-8aa20296-debe-499e-ab77-fbf1c710f717.png">

Unfortunately we don't have a way to update the positioning of the icons of text. There are a couple storybooks that change the font and they all have this problem.

> Also seeing the old hover color, I wonder if we can get the hover to match the Sage leaf color? <img alt="CleanShot 2023-02-14 at 11 15 04@2x" width="239" src="https://user-images.githubusercontent.com/791670/218865587-66adc658-9141-461a-98da-5f2dcb0ed2d2.png">

Yep, I'll remove the added font